pages/base.py

```diff
@@ -64,22 +64,22 @@ def create_new_user(self, email, password):
 
         WebDriverWait(self.selenium, 20).until(lambda s: self.is_user_loggedin)
```

Comment:
Should this also be changed to `WebDriverWait(self.selenium, self.timeout).until(lambda s: self.is_user_loggedin)`?